docs/plugins/components.md

```diff
@@ -0,0 +1,842 @@
+---
+sidebar_position: 5
+title: Components
+description: Documented components for plugins
+---
+
+# Components
+
+:::caution
+
+Any components below whose name ends in `Item` already has a `FormItem` (or equivalent) wrapper. You
+should not use this on those components.
+
+:::
+
+## Input components
+
+All input components ending with `Item` have their original counterpart, and differ in some props:
+
+| Name        | Type                | Default | Description                                      |
+| ----------- | ------------------- | ------- | ------------------------------------------------ |
+| `children?` | React.ReactNode     |         | Wrapper component title                          |
+| `note?`     | string              |         | Description of what the input component is about |
+| `style?`    | React.CSSProperties |         | Wrapper component style                          |
```

Comment:
Link to https://reactjs.org/docs/dom-elements.html#style to explain how this property works

Reply:
Done. I converted the page file to `mdx` so that we can create reusable components in markdown.